src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -27,25 +27,40 @@
 from pybuilder.errors import BuildFailedException
 from pybuilder.utils import assert_can_execute
 from pybuilder.utils import execute_command
+from pybuilder import scaffolding as SCAFFOLDING
+from pybuilder.core import NAME_ATTRIBUTE
 
-__author__ = 'Thomas Prebble'
+__author__ = 'Thomas Prebble', 'Marcel Wolf'
 
 
 use_plugin("core")
 
 
-DEFAULT_SPHINX_BUILDER = "html"
-DEFAULT_SPHINX_SOURCE_DIR = ""
-DEFAULT_SPHINX_OUTPUT_DIR = "_build/"
+DEFAULT_SPHINX_OUTPUT_DIR = SCAFFOLDING.DEFAULT_DOCS_DIRECTORY + "/_build/"
+
+SPHINX_DOC_AUTHOR = "doc_author"
+SPHINX_DOC_BUILDER = "html"
+SPHINX_PROJECT_NAME = "myproject"
+SPHINX_PROJECT_VERSION = "0.1"
```

Comment:
Use `project.version`? It can't be static then, but that's okay. We can set it in `initialize_sphinx_plugin` for the default